kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -23,16 +23,24 @@ The script assumes that either the image, `container-registry.oracle.com/middlew
 
 ```
 $ ./create-rcu-schema.sh -h
-usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <rcuType> -d <dburl> -i <image> -s <dockerstore> [-h]
-  -s RCU Schema Prefix (required)
+usage: ./create-rcu-schema.sh -s <schemaPrefix> -t <schemaType> -d <dburl> -i <image> -p <docker-store> -n <nameSpace> -q <sysPassword> -r <schemaPassword>  -o <rcuOutputDir>  [-h]
```

Comment:
i assume all of these issues are in the actual script too, and need to be fixed there...